lib/signs_ui/config/state.ex

```diff
@@ -5,6 +5,8 @@ defmodule SignsUi.Config.State do
   use GenServer
   require Logger
 
+  alias SignsUi.Config
+
```

Comment:
If you're re-adding this, you can also undo the other change in 1b3654f that changed `Config.Request.get_signs()` to `SignsUi.Config.Request.get_signs()`.

Reply:
Good idea.

Reply:
Okay, amended that commit as you suggested. Actually, since we're amending commits like this, I'm going to figure out when/where/how dialyzer broke and fix that commit rather than the fix-up commit at the end.

Reply:
Hm, this is proving tricker than I thought, as amending the commit where the dialyzer failure is introduced has cascading merge conflicts later on.

Unfortunately, I think that means I need to squash all these as I merge. But I guess that's not a problem since conceptually it is just one big renaming job anyway.